src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -161,8 +161,8 @@ public ItemGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		onDroppedByPlayer = new ProcedureSelector(this.withEntry("item/on_dropped"), mcreator,
 				L10N.t("elementgui.item.event_on_dropped"),
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity/itemstack:itemstack"));
-		onEaten = new ProcedureSelector(this.withEntry("item/when_eaten"), mcreator,
-				L10N.t("elementgui.food.event_on_eaten"),
+		onFinishUsingItem = new ProcedureSelector(this.withEntry("item/when_stopped_using"), mcreator,
+				L10N.t("trigger.player_useitem_finish"),

```

Comment:
`trigger.*` strings are used for procedure triggers (inside Blockly), you should switch to some other key here.

Reply:
Blockly triggers use `blockly.trigger.*`. Also, even if it uses a Blockly trigger text, the line describes perfectly this trigger.

Reply:
It should follow elementgui.item. naming convention, though @Goldorion